src/vivarium_gates_nutrition_optimization_child/constants/paths.py

```diff
@@ -15,12 +15,12 @@
 MMS_GA_SHIFT_1_DATA_DIR = RAW_DATA_DIR / "mms_gestational_age_shifts/shift1"
 MMS_GA_SHIFT_2_DATA_DIR = RAW_DATA_DIR / "mms_gestational_age_shifts/shift2"
 SQLNS_RISK_RATIOS = RAW_DATA_DIR / "2021_sqlns_effects_by_location.csv"
-WASTING_TRANSITIONS_DATA_DIR = RAW_DATA_DIR / "wasting_transition_rates/2021"
+WASTING_TRANSITIONS_DATA_DIR = RAW_DATA_DIR / "wasting_transition_rates/subnational"
 WASTING_TREATMENT_PARAMETERS_DIR = RAW_DATA_DIR / "wasting_treatment_parameters"
-WASTING_RELATIVE_RISKS = RAW_DATA_DIR / "wasting_rrs_with_subcategories.csv"
-PROBABILITIES_OF_WORSE_MAM_EXPOSURE = RAW_DATA_DIR / "worse_exp_frac.csv"
-UNDERWEIGHT_CONDITIONAL_DISTRIBUTIONS = RAW_DATA_DIR / "lookup.csv"
-CGF_PAFS = RAW_DATA_DIR / "cgf_pafs.csv"
+WASTING_RELATIVE_RISKS = "/mnt/team/simulation_science/pub/models/vivarium_gates_nutrition_optimization_child/raw_data/wasting_rrs_with_subcategories_only_locations.csv"
```

Comment:
So these data files ended up being really big when we made them into subnational specific files. Therefore we decided to put them on the cluster, NOT on GitHub since they were over the recommended file size for GtiHub. The other `RAW_DATA_DIR` is a folder within the engineering repository. That's why the address is different. Is there another way I should do this?